gigadb/app/tools/excel-spreadsheet-uploader/template/UPDATE_INSTRUCTIONS.md

```diff
@@ -0,0 +1,46 @@
+
+## Using Github Desktop
+
+1. Assuming you have already cloned Gigascience's gigadb-website Github project in Github Desktop
+2. Open Github Desktop, and ensure that:
+    * The `gigadb-website` project is selected as the "Current repository` in the toolbar
+    * The `develop` branch is selected as the "Current branch" in the toolbar
+    * Click "Fetch origin" to ensure you have the latest state of the truth from the server
+    * The "Changes" column on the left should be empty
+3. Identify where Github Desktop has clone the repository on your computer and navigate to it.
+4. On your computer, make the change you want to the file in `gigadb/app/tools/excel-spreadsheet-uploader/template`
+5. Then return to Github Desktop, you should see that the "Changes" column on the left is not empty and reflects the changes your made
+6. Create a new branch by clicking on the drop-down menu right of "Current branch" in the toolbar
+7. Click on "New branch", choose a meaningful name (e.g: fix-doi-hint), and click "Create branch"
+8. Then make sure that out of the two options shown, you select "Bring my changes to (name of the branch just created)"
+9. Click "Switch branch", "Current branch" in the toolbar should now show the name of the branch with your changes
+10. It is time to commit the changes by adding a summary and description to the commit message form at the bottom left, and then click "commit to (name of the just created branch)"
+11. The main pane will show a list of what to do next. Choose the top one (already highlighted) "Publish your branch" by clicking the "Publish branch" button
+12. The top option in the main pane is highlighted and says "Preview the pull request from you current branch", and has a button "Preview Pull Request" that you click
+13. A preview pane will pop up, check that:
+    *  it says "commit into base:develop from (name of the branch with your changes)"
+    *  the main pane contain the change you want to publish and be reviewed
+    *  At the bottom left, it should have a green tick "Able to merge"
+14. If all good, you can click "Create Pull Request"
+15. Your web browser will open on a new Pull request form on the Gigascience's gigadb-website repository
+16. Fill in the form as instructed inline
+17. Click on "Create Pull Request" (if instead you see "Draft Pull Request", click the drop-down menu to bring the other option)
+18. In the "Projects" section in the right-hand sidebar, add the pull request to the current sprint project
```

Comment:
Might be better re-written like this:

18. Add the pull request to the current sprint project by pasting its link into a new note in the Tasks To Do column. Sprint projects are listed in [Projects (classic)](https://github.com/orgs/gigascience/projects?type=classic).